src/i18n/en.json

```diff
@@ -113,16 +139,97 @@
         "charisma-full": "Charisma"
       },
       "Combat": {
+        "DamageBonus": "Damage Bonus",
         "StrDexMinsNotMetTip": "Character does not meet STR and/or DEX minimums for this weapon.  Use at half skill chance.",
         "StrMinNotMetTip": "Character does not meet STR minimum for this weapon. Use at half skill chance.",
-        "DexMinNotMetTip": "Character does not meet DEX minimum for this weapon. STR over STR minimum can make up for this at two points of STR for one point of DEX, or use at half skill chance."
+        "DexMinNotMetTip": "Character does not meet DEX minimum for this weapon. STR over STR minimum can make up for this at two points of STR for one point of DEX, or use at half skill chance.",
+        "Weapon": "Weapon",
+        "HP": "HP",
+        "Chance": "Chance",
+        "Damage": "Damage",
+        "SR": "SR",
+        "Dodge": "Dodge",
+        "SpiritCombat": "Spirit Combat",
+        "SpiritCombatDamage": "Spirit Combat Damage"
       },
       "Gear": {
+        "ViewByItemType": "View by Item Type",
+        "ViewByLocation": "View by Locaion",
+        "Gear": "Gear",
+        "Armor": "Armor",
+        "Consumables": "Consumables",
+        "Currency": "Currency",
+        "WeaponsAndShields": "Weapons & Shields",
         "CurrencyConversionTipOver1": "One {name} is worth {value} Lunars.",
         "CurrencyConversionTipLunar": "Lunars are the standard basis of currency.",
-        "CurrencyConversionTipUnder1": "One Lunar is worth {value} {name}s."
+        "CurrencyConversionTipUnder1": "One Lunar is worth {value} {name}s.",
+        "Name": "Name",
+        "EncumbranceAbbr": "ENC",
```

Comment:
Should this be moved to Game? there is both `Actor.Gear.EncumbranceAbbr`, `Actor.Attributes.EncumbranceAbbr` and `Item.Gear.EncumbranceAbbr`.  Not sure though, and that would maybe imply a lot of other stuff should be moved "out". We could also live with multiple translations of the same thing...

Reply:
In general I was trying NOT to de-dupe translations for two reasons. First, it was confusing to remember where I put the "common" translations.  Second, I figured there might be some weird circumstance where the duplication actually needed different translations in different places in some other language because of the context.  If we don't think the second reason is a problem, then I think it was still easier to put the translations "in place" and then, seeing the structure we created, decide on how to create "common" keys.

Reply:
Of course there were definitely exceptions where I felt more confident that one translation would always work and where a clear way of putting them in a common place suggested itself.  Again, I thought it better to just get it done and consider re-organization when we know what we have to re-organize.

Reply:
Which is to say I didn't change this.